### DISCUSSION TITLE
Update 4.6.0

### DISCUSSION THREAD

**Discussion by bluecaret** [bluecaret/carettab#383] [! News and Announcements]
# 4.6.0 (10/04/2024)

## New features
- Wallpapers can now be any file size! You will still receive a warning message if the file is large saying that it can affect performance, but the wallpaper will apply regardless.
- Added additional "special links" to Quick Links include downloads page, extensions page, browser inspect, browser flags, browser settings, browser about.

## Fixes and other changes
- Changed weather forecast days to be capped at 3 days instead of 5. This allows me to use a lower tier plan for the weather API so I don't have to eat the costs for the data since I'm not making money on CaretTab anymore.
- Changed general API used to reduce costs
- Adjusted some links to CaretTab's homepage.
- Fixed: Quick links set to a vertical list had an issue where the last link in the list would always display in the more menu even if it has room.
- Translation fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/bluecaret/carettab/releases/tag/v4.6.0'>Update 4.6.0</a>.</em>

**Comment by ryanbuening** [comment]
@bluecaret - fyi - I received the notification below in Edge when upgrading to 4.6.0. Wasn't sure if that was to be expected, but just wanted to share.

![image](https://github.com/user-attachments/assets/253cae68-bbe1-4eae-9f84-c301439beb75)

**Reply by bluecaret** [reply]
Yes this was expected. I had to change the URLs needed for accessing weather and wallpaper data, and those URLs need to be listed as permissions which cause this message. I haven't found a good way to inform about these types of changes yet which is unfortunate with how sketchy it looks. Thanks Google.. Anyway thank you for letting me know!
